File: Up/app/src/main/java/com/product/dstavrov/up/Helpers/Day.java
package com.product.dstavrov.up.Helpers;

import java.util.HashMap;
import java.util.Map;

/**
 * Created by vicru on 20.03.2018.
 */

public class Day {
    private String name;
    private Map<String, Subject> rasp;
    public Day(){
        rasp = new HashMap<String, Subject>();
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void add_subject(String time, Subject subject){//time format (7_30-9_10)
        rasp.put(time, subject);
    }

    public Map<String, Subject> getRasp() {
        return rasp;
    }
}

File: Up_wpf/Up_wpf/Day.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Up_wpf
{
    public class Day
    {
        private string name;
        private Dictionary<string, Subject> rasp;
        public Day()
        {
            rasp = new Dictionary<string, Subject>();
        }

        public string getName()
        {
            return name;
        }
        public void setName(string name)
        {
            this.name = name;
        }

        public void add_subject(string time, Subject subject)
        {//time format (7_30-9_10)
            rasp.Add(time, subject);
        }

        public Dictionary<string, Subject>  getRasp()
        {
            return rasp;
        }
    }
}
